youtube_dl/extractor/markiza.py

```diff
@@ -26,54 +28,89 @@ class MarkizaIE(InfoExtractor):
             'duration': 2760,
         },
     }, {
-        'url': 'http://videoarchiv.markiza.sk/video/televizne-noviny/televizne-noviny/85430_televizne-noviny',
+        'url': ' https://videoarchiv.markiza.sk/video/laska-na-prenajom/epizoda/58779-seria-1-epizoda-14',
         'info_dict': {
             'id': '85430',
             'title': 'Televízne noviny',
         },
         'playlist_count': 23,
     }, {
-        'url': 'http://videoarchiv.markiza.sk/video/oteckovia/84723',
+        'url': 'https://videoarchiv.markiza.sk/video/oteckovia/84723',
         'only_matching': True,
     }, {
-        'url': 'http://videoarchiv.markiza.sk/video/84723',
+        'url': 'https://videoarchiv.markiza.sk/video/84723',
         'only_matching': True,
     }, {
-        'url': 'http://videoarchiv.markiza.sk/video/filmy/85190_kamenak',
+        'url': 'https://videoarchiv.markiza.sk/video/filmy/85190_kamenak',
         'only_matching': True,
     }, {
-        'url': 'http://videoarchiv.markiza.sk/video/reflex/zo-zakulisia/84651_pribeh-alzbetky',
+        'url': 'https://videoarchiv.markiza.sk/video/reflex/zo-zakulisia/84651_pribeh-alzbetky',
         'only_matching': True,
     }, {
-        'url': 'http://videoarchiv.markiza.sk/embed/85295',
+        'url': 'https://videoarchiv.markiza.sk/embed/85295',
         'only_matching': True,
     }]
 
     def _real_extract(self, url):
         video_id = self._match_id(url)
+        webpage = self._download_webpage(url, video_id)
+        embed = self._search_regex(
+            r'<iframe src="(https:\/\/media.*?)" style="',
+            webpage, 'embed', fatal=False)
+        webpage = self._download_webpage(embed, video_id)
+        data = self._search_regex(
+            r'processAdTagModifier\((\{.*)\), {"video":',
+            webpage, 'embed', fatal=False)
+        data2 = self._search_regex(
+            r'processAdTagModifier\(\{.*\), ({"video":.*)\);',
+            webpage, 'embed', fatal=False)
+        data = re.sub('\\\\/', '/', data)
+        data2 = re.sub('\\\\/', '/', data2)
+        info = json.loads(data)
```

Comment:
Here and in the next line consider `self._parse_json()` instead of calling `json.loads()` directly.